test/built-ins/Map/prototype/has/returns-boolean.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
```

Comment:
I think this test is made redundant by test/built-ins/Map/prototype/has/return-false-different-key-types.js and test/built-ins/Map/prototype/has/return-true-different-key-types.js. Those other two seem preferable because they have less dependencies (they don't need `Map.prototype.clear` or `Map.prototype.delete`)